UPGRADE-4.0.md

```diff
@@ -0,0 +1,9 @@
+UPGRADE FROM 3.x to 4.0
+=======================
+
+### DoctrineBridge and Validator
+
+ * If you were using the `StaticMethodLoader` to create `UniqueEntity`
```

Comment:
PS: The use case would still be possible (and the deprecation still required) for custom constraint loaders, just thought about that...